android/src/main/java/com/mattermost/networkclient/helpers/Helper.kt

```diff
@@ -137,6 +137,51 @@ fun OkHttpClient.Builder.parseOptions(options: ReadableMap?, request: Request.Bu
         }
     }
 
+    // Request Adapter Config
+    if(options.hasKey("requestAdapterConfiguration")){
+        val requestAdapterConfiguration = options.getMap("requestAdapterConfiguration")!!;
+        if(requestAdapterConfiguration.hasKey("bearerAuthTokenResponseHeader")){
+            val bearerAuthTokenResponseHeader = requestAdapterConfiguration.getString("bearerAuthTokenResponseHeader")!!
+            this.addInterceptor(BearerTokenInterceptor(bearerAuthTokenResponseHeader))
+        }
+    }
+
+    // Client Cert Config
+    if(options.hasKey("clientP12Configuration")){
+        val clientP12Configuration = options.getMap("clientP12Configuration")!!;
+
+        // Get our Cert
+        val path = clientP12Configuration.getString("path")!!;
+        val password = if(clientP12Configuration.hasKey("password")) {
+            clientP12Configuration.getString("password")!!
+        } else {
+            ""
+        }
+
+        // Save P12 into KeyStore
+        val aliases = getAndSaveCertsFromP12(path, password)
+
+        // Use it for our Client
+        val keyStore = KeyStore.getInstance("AndroidKeyStore")
+        keyStore.load(null, null)
```

Comment:
What does this do?

Reply:
It just "loads" the keystore; https://developer.android.com/training/articles/keystore#WorkingWithKeyStoreEntries

Reply:
Do we need to load it for every client that gets built? Does this have any perf implication?